src/Enhavo/Bundle/SettingBundle/Form/Type/SettingType.php

```diff
@@ -45,9 +84,67 @@ public function getName()
         return 'enhavo_setting_setting';
     }
 
-    public function getFormType()
+    public function getFormType() // TODO why?
```

Comment:
I think we don't use this function anywhere, please check ``composeCreateForm`` and ``composeUpdateForm`` also.